src/Microsoft.AspNetCore.Razor.Tasks/Microsoft.AspNetCore.Razor.Tasks.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- This needs to support desktop and CoreCLR to run in msbuild -->
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
`net461` should suffice

Reply:
Are we doing that? I'm being consistent with other stuff. Everything else in Razour is net46